pkg/sdk/resource_monitors_integration_test.go

```diff
@@ -0,0 +1,246 @@
+package sdk
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_ResourceMonitorsShow(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	resourceMonitorTest, resourceMonitorCleanup := createResourceMonitor(t, client)
+	t.Cleanup(resourceMonitorCleanup)
+
+	_, resourceMonitorCleanup2 := createResourceMonitor(t, client)
```

Comment:
why do you create two resource montiors if you only use one of them?

Reply:
probably copy-paste from other resource, fixed

Reply:
probably copy-paste from other resource, fixed